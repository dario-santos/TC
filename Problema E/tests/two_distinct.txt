2
S -> a
A -> _
