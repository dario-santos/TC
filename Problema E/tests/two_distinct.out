NULL(S) = False
NULL(A) = True
FIRST(S) = a
FIRST(A) =
FOLLOW(S) = #
FOLLOW(A) =
